src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/CallbackParams.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks;
+
+import lombok.Builder;
+import lombok.Getter;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+
+import java.util.Map;
+
+@Builder
+@Getter
+public class CallbackParams {
+    private CallbackType type;
+    private CallbackRequest request;
+    private Map<Params, Object> params;
+
+    public enum Params {
```

Comment:
This I don't like much, but at the same time each callback might or might not need params specific params and I needed to find a way to pass them in. There are better ways (ie. Joshua Bloch's typesafe hetereogeneous container) but I feel they are a bit over complicated for what we need. We can always refactor if we realise this is getting ugly